en/sensor/airspeed.md

```diff
@@ -31,7 +31,7 @@ Additionally, the [Avionics Anonymous Air Data Computer](https://www.tindie.com/
 
 ### Enable Airspeed Sensors
 
-Unlike most other sensor drivers, the airspeed sensor drivers are not automatically started.
+Unlike most other sensor drivers, the airspeed sensor drivers are not automatically started.This feature is added after v1.13.0
```

Comment:
Are you saying that airspeed sensors ARE started automatically after v1.13? 

I thought in main branch now we weren't auto-starting them?

Reply:
No, i say the opposite.I am saying, airspeed sensors started automatically before v1.13.0

After some development in v1.13.0 PX4 hasn't start automatically airspeed sensor.

Reply:
Just look at this link
`https://github.com/PX4/PX4-Autopilot/commit/3f172dbfa7b35e0be4ce7cb6544b33a35ebd9531`